.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+node_js:
+- 10
+services:
+  - docker
+install: npm install
+
+stages:
+  - lint
+  - name: build docker image
+    if: branch != master || type != push
```

Comment:
Why you need this condition?
Every branch will build the image, and if you push the master he will build the image with this step deploy in another stage.

Reply:
Each stage runs in a totally separated environment, so if I build the image in "build docker image" stage, the image does not exist in "deploy" stage

Reply:
So where are the images being saved? Because if they are not, why we would build it every push (without pushing to dockerhub)?

Reply:
We've already talked about it... It's to make sure the build passes.